### PR TITLE
first draft of data road map

### PR DESCRIPTION
**Pull request by ilanman**
Just went through a road map exercise at work and tried to distill most important elements. Trying to keep it short, but let me know:

1. If I can/should shorten it
2. If you have any suggestions.

Would be great to post Sunday Jan 6th when folks are back from vacation.

### CODE REVIEW

File: content/post/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+<li> Communicate your team's priorities to leaders across the organization, e.g., When your team is focused on data infrastructure for all of Q1, stakeholders may be left unsatisfied. It's helpful to identify how each of the various Data infrastructure projects (i.e., adding testing and error alerting, data re-model, `dbt`) will ultimately support stakeholder initiatives slated for Q2 and beyond.
+<li> Clarify (and hopefully excite) your team for the year ahead. The road map is an opportunity to be ambitious and set up your team to think about what their year can conceivably look like.
+<li> Help defend head count for future hiring needs and also focus your hiring search, e.g., When Finance asks you why you need 7 more head count through Q3, a road map will allow you to identify what the break down is (junior vs. senior, analyst vs. data scientist) and to what projects they would be assigned. 
+ 
+### Identify audience for such a document
```
Comment (mikekaminsky) [inline content/post/roadmap.md]: "the audience"

Comment (mikekaminsky) [reply content/post/roadmap.md]: Maybe just rewrite to "Identifying the audience for the roadmap"

File: content/post/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Ideally everyone in the organization has access to this road map. My preferred medium for communicating my road map is presentation. I can inject visuals if needed, color code where appropriate and use the natural aspect of "one topic per slide" to help navigate.
```
Comment (mikekaminsky) [inline content/post/roadmap.md]: >is presentation.

could be clarified to 

> a slide deck with an in-person presentation.

File: content/post/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+<li> Help defend head count for future hiring needs and also focus your hiring search, e.g., When Finance asks you why you need 7 more head count through Q3, a road map will allow you to identify what the break down is (junior vs. senior, analyst vs. data scientist) and to what projects they would be assigned. 
+ 
+### Identify audience for such a document
+
+Ideally everyone in the organization has access to this road map. My preferred medium for communicating my road map is presentation. I can inject visuals if needed, color code where appropriate and use the natural aspect of "one topic per slide" to help navigate. 
+
+While everyone should have access, you don't need to make it digestible to everyone in the organization. You key audience should be leadership, relevant cross-functional partners and your team. That will help guide the level of detail required. I also like to present my road map to stakeholders, which allows me to keep it high level in the presentation and speak to it when necessary.
```
Comment (mikekaminsky) [inline content/post/roadmap.md]: > I also like to present my road map to stakeholders, which allows me to keep it high level in the presentation and speak to it when necessary.

I'm not sure I follow what the 'it' in this sentence is ... is it the roadmap? or specific concerns?

File: content/post/roadmap.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+### Relate to organization’s mission and values
```
Comment (mikekaminsky) [inline content/post/roadmap.md]: " the organization"

File: content/post/roadmap.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+As a data team its incumbent upon us to ensure that decisions are tied to organizational objectives. This is part of thinking holistically about your business. In that vein, you should strive to tie each of your high level strategic goals to the organization's mission and/or objectives. It's a helpful exercise to tie each project, however indirectly, to the company's strategic objectives, even if you don't end up documenting that in the final road map.
```
Comment (mikekaminsky) [inline content/post/roadmap.md]: > As a data team, it's

File: content/post/roadmap.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+You may find it more natural to split your road map by the various teams you support, e.g. Marketing, Product, Sales, etc... I prefer the former because it captures both technical differences that are relevant to the team (e.g. data analysts will skew heavily towards BI, data engineers towards data infrastructure, etc...) and "Analytics and Strategic projects" encapsulate projects that are cross functional (e.g. Marketing, Sales, Finance).
+
+### Sync with functional leaders 
+
+After putting down some projects that fit into the relevant areas of focus, set up time with key functional leaders and get their input. Hopefully you sequence this exercise at a time in which they are also thinking about their goals for the year. Have them help you fill in blind spots and loosely prioritize their projects. You may know of 4 Operations projects that are critical for 2019, but the COO may have a different idea on what the sequencing should be. Don't get bogged down by timing at this point, but getting the order correct is very helpful.
```
Comment (mikekaminsky) [inline content/post/roadmap.md]: > getting the order correct

Does this refer to chronology or priority?

File: content/post/roadmap.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+Functional leaders will also be helpful in identifying cross team dependencies. You may have grand plans for implementing [Airflow](https://airflow.apache.org/), but if you lack Data Engineering resources, you may have to rethink your plan. Similarly you may have an idea for a great data-driven product feature, but if Product isn't prioritizing that in their roadmap, it may drop to the bottom of your priority, even though you think it's a valuable feature.
+
+### Put stakes in the ground 
+
+Once you're confident that you have covered the most important projects for the year, write down estimated timing for completion and key stakeholders impacted. For example, under "Analytics" you may have the bullet `Establish a multi-touch attribution model // Q2 // Marketing`.
+
+After you do this across the board, you'll have a good sense of how projects are being sequenced, if you're front-loading projects (too many in Q1/Q2), and which functions aren't being supported (or are being overly represented). This functions as another sense check for how you're orienting your year and helps to communicate and set expectations across the organization.
```
Comment (mikekaminsky) [inline content/post/roadmap.md]: The re-use of "functions" in two different sentences back-to-back here is a little confusing

Comment (mikekaminsky) [inline content/post/roadmap.md]: At first I read this as a run-on that was hard to parse:

> After you do this across the board, you'll have a good sense of how projects are being sequenced, if you're front-loading projects (too many in Q1/Q2), and which functions aren't being supported (or are being overly represented).

I'd consider either adding punctuation or breaking it up into two sentences.

File: content/post/roadmap.md (hunk 3 above)
Comment (ilanman) [reply content/post/roadmap.md]: Agreed. Tried to add clarity.

File: content/post/roadmap.md (hunk 6 above)
Comment (ilanman) [reply content/post/roadmap.md]: both? Tried to clarify.

File: content/post/roadmap.md (hunk 7 above)
Comment (ilanman) [reply content/post/roadmap.md]: How about bullets?

File: content/post/roadmap.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+You may end up tying your work back to, say, 2 of your organization's 3 strategic goals for the year. That's not in and of itself bad, but should be called out. This exercise reduces blind spots and forces you to think about how you can support all aspects of the mission.
+
+### Set up the scaffolding
+
+How do you start? After outlining our Data team mission and values (distinct from the broader company) I like to make explicit what areas of focus we're responsible for. For us, that's Data infrastructure, BI, analytics and strategic projects, Data Science and Clinical Data products. This helps to compartmentalize projects, especially those that are cross functional.
+
+You may find it more natural to split your road map by the various teams you support, e.g. Marketing, Product, Sales, etc... I prefer the former because it captures both technical differences that are relevant to the team (e.g. data analysts will skew heavily towards BI, data engineers towards data infrastructure, etc...) and "Analytics and Strategic projects" encapsulate projects that are cross functional (e.g. Marketing, Sales, Finance).
```
Comment (sbreitenother) [inline content/post/roadmap.md]: e.g. x3 -> e.g.,

File: content/post/roadmap.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+### Sync with functional leaders 
+
+After putting down some projects that fit into the relevant areas of focus, set up time with key functional leaders and get their input. Hopefully you sequence this exercise at a time in which they are also thinking about their goals for the year. Have them help you fill in blind spots and loosely prioritize their projects. You may know of four Operations projects that are critical for 2019, but the COO may have a different idea on what the sequencing should be. Don't get bogged down by timing at this point, but getting a rank-ordered list of projects - at least within each functional stakeholder - is very helpful. For example, it may be too early to understand if Finance's pricing model should be prioritized over the Customer Support team's call queue algorithm, but its helpful to identify if automating Finance's monthly billing report is more important than their pricing model.
```
Comment (sbreitenother) [inline content/post/roadmap.md]: Might be a good spot to link to your sprint planning meeting  post.

File: content/post/roadmap.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+Functional leaders will also be helpful in identifying cross team dependencies. You may have grand plans for implementing [Airflow](https://airflow.apache.org/), but if you lack Data Engineering resources, you may have to rethink your plan. Similarly you may have an idea for a great data-driven product feature, but if Product isn't prioritizing that in their roadmap, it may drop to the bottom of your priority, even though you think it's a valuable feature.
+
+### Put stakes in the ground 
+
```
Comment (sbreitenother) [inline content/post/roadmap.md]: Do you recommend attempting to do some sort of ROI calculation or avoiding ROI all together?

Comment (ilanman) [reply content/post/roadmap.md]: Two thoughts:
1. A rank-ordered list will hopefully have some approximation of ROI embedded within in. 
2. Any level of ROI detail greater than the rank-ordered list is probably unnecessary and ultimately inaccurate at this stage. For sake of expediency and rallying around projects/objectives, probably best to save ROI calcs for when you're closer to resource allocation time.
